src/main/res/values/strings.xml

```diff
@@ -120,47 +120,47 @@
   <string name="saving_details">Saving details</string>
   <string name="screenshots">Screenshots</string>
   <string name="search">Search</string>
-  <string name="select_the_mirror">Select the mirror</string>
-  <string name="show_more">Show more</string>
-  <string name="show_older_releases">Show older releases</string>
+  <string name="select_the_mirror">Select a mirror</string>
+  <string name="show_more">More</string>
+  <string name="show_older_releases">Show older versions</string>
   <string name="skip">Skip</string>
   <string name="socks_proxy">SOCKS proxy</string>
-  <string name="sort_order">Sort order</string>
+  <string name="sort_order">Sorting order</string>
   <string name="source_code">Source code</string>
   <string name="suggested">Suggested</string>
   <string name="sync_repositories">Sync repositories</string>
-  <string name="sync_repositories_automatically">Sync repositories automatically</string>
-  <string name="syncing">Syncing</string>
-  <string name="syncing_format">Syncing %s</string>
+  <string name="sync_repositories_automatically">Automatically</string>
+  <string name="syncing">Syncing…</string>
+  <string name="syncing_format">Syncing %s…</string>
```

Comment:
Ellipses are not necessary, since progress bar is displayed.

Reply:
I was meaning to ask what the first "Syncing" is. Verb or noun?
Also, why is it not "syncronizing" or "syncronising"? (Just curious.)

Reply:
Both are nouns. "Syncing" is used for notification group title (for Android 8+). "Syncing %s" is used in notification title, like "Syncing F-Droid".

It seems that shorting to "sync" is pretty common in English. As for notification title, it should not be very long (syncronizing is really a long word).